From 89dc13fb4d2ab69b4e324a8c2128471bb1ff495c Mon Sep 17 00:00:00 2001
From: Samuel Verschelde <stormi-xcp@ylix.fr>
Date: Tue, 2 Aug 2022 17:05:32 +0200
Subject: [PATCH 18/18] Add support for RHEL 9 and derivatives in install.sh

Signed-off-by: Samuel Verschelde <stormi-xcp@ylix.fr>
---
 mk/install.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/mk/install.sh b/mk/install.sh
index 44c6f7c..ac75e0f 100755
--- a/mk/install.sh
+++ b/mk/install.sh
@@ -138,6 +138,9 @@ select_pkgs_rhel()
     rhel8|oracle8|centos8|rocky8|cloudlinux8|almalinux8)
 	    # No additional kernel package
 	    ;;
+	rhel9|oracle9|centos9|rocky9|cloudlinux9|almalinux9)
+	    # No additional kernel package
+	    ;;
 	sangoma*)
 	    # No additional kernel package
 	    ;;
-- 
2.30.2

